src/client/components/region-banner-component/banner.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import { Card, Container } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+
+import 'client/components/region-banner-component/index.scss';
+import birka from 'client/components/region-banner-component/img/birka.svg';
+
+export class BannerComponent extends Component {
+    constructor(props) {
```

Comment:
Since you aren't initializing or binding anything, this constructor is redundant.